kahuna/public/js/search/syntax/syntax.html

```diff
@@ -123,7 +123,7 @@ <h3 class="advanced-search-help__section__sub-heading">How to filter by Usage hi
                 <gr-chip-example gr-filter-field="usages@platform" gr-example-search="print"></gr-chip-example>
             </dt>
             <dd class="advanced-search-example__explanation">
-                Filter by print or digital usage.
+                Filter by print, digital or download usage.
```

Comment:
Since this is dependent on config (that can be OFF and `download` platform won’t be available), maybe better to say `Filter by print, digital or other usage.`? Not a biggie, obvs.

Reply:
Thanks @paperboyo for the suggestion. This is now resolved.